java-utils/mvn_build.py

```diff
@@ -188,12 +190,14 @@ def goal_callback(option, opt_str, value, parser):
     p = subprocess.Popen(" ".join(mvn_args), shell=True, env=os.environ)
     p.wait()
 
-    subprocess.Popen("""
-        if [ -f .xmvn-builddep ]; then
-            echo -----BEGIN MAVEN BUILD DEPENDENCIES-----
-            gzip -9nc <.xmvn-builddep | base64
-            echo -----END MAVEN BUILD DEPENDENCIES-----
-        fi
-        """, shell=True, env=os.environ).wait()
-
-    sys.exit(p.returncode)
+    fname = ".xmvn-builddep"
+    if (os.path.exists(fname) and os.path.isfile(fname)):
```

Comment:
If a file is regular file then it must exist, so `os.path.isfile(fname)` should be enough.